tests/unit/amo/components/TestAddonTitle.js

```diff
@@ -26,54 +26,64 @@ describe(__filename, () => {
       addon: createInternalAddon({ ...fakeAddon, name }),
     });
 
-    expect(root.html()).toContain(name);
+    expect(root).toIncludeText(name);
   });
 
   it('renders a single author', () => {
-    const authorUrl = 'http://olympia.test/en-US/firefox/user/krupa/';
+    const author = {
+      ...fakeAddon.authors[0],
+      username: 'some-username',
+    };
+
     const root = render({
-      addon: createInternalAddon({
-        ...fakeAddon,
-        authors: [
-          {
-            name: 'Krupa',
-            url: authorUrl,
-          },
-        ],
-      }),
+      addon: createInternalAddon({ ...fakeAddon, authors: [author] }),
     });
 
-    expect(root.html()).toContain('Krupa');
-    expect(root.html()).toContain(authorUrl);
+    expect(root.find(Link)).toHaveLength(1);
+    expect(root.find(Link)).toHaveProp('children', author.name);
+    expect(root.find(Link)).toHaveProp('to', `/user/${author.username}/`);
   });
 
   it('renders multiple authors', () => {
+    const author1 = {
+      ...fakeAddon.authors[0],
+      name: 'Author 1',
+      username: 'author-1',
+    };
+    const author2 = {
+      ...fakeAddon.authors[0],
+      name: 'Author 2',
+      username: 'author-2',
+    };
+
     const root = render({
-      addon: createInternalAddon({
-        ...fakeAddon,
-        authors: [
-          {
-            name: 'Krupa',
-            url: 'http://olympia.test/en-US/firefox/user/krupa/',
-          },
-          {
-            name: 'Fligtar',
-            url: 'http://olympia.test/en-US/firefox/user/fligtar/',
-          },
-        ],
-      }),
+      addon: createInternalAddon({ ...fakeAddon, authors: [author1, author2] }),
     });
 
-    expect(root.html()).toContain('Krupa');
-    expect(root.html()).toContain('Fligtar');
-    expect(root.render().find('a')).toHaveLength(2);
+    expect(root.find(Link)).toHaveLength(2);
+    expect(root.find(Link).at(0)).toHaveProp('children', author1.name);
+    expect(root.find(Link).at(0)).toHaveProp(
+      'to',
+      `/user/${author1.username}/`,
+    );
+    expect(root.find(Link).at(1)).toHaveProp('children', author2.name);
+    expect(root.find(Link).at(1)).toHaveProp(
+      'to',
+      `/user/${author2.username}/`,
+    );
+    // This assertion is used to make sure we add commas between each author
+    // link/name.
+    expect(root).toIncludeText(
```

Comment:
This seems a bit fragile. I think we try to avoid using enzyme output in assertions. Could you just check that there is a `, ` in the string somewhere? Maybe check that there is only one?

Reply:
nevermind, I found a better way!